src/jinja2/utils.py

```diff
@@ -19,7 +19,11 @@
 F = t.TypeVar("F", bound=t.Callable[..., t.Any])
 
 # special singleton representing missing values for the runtime
-missing: t.Any = type("MissingType", (), {"__repr__": lambda x: "missing"})()
+missing: t.Any = type(
+    "MissingType",
+    (),
+    {"__repr__": lambda x: "missing", "__reduce__": lambda x: "missing"},
+)()
```

Comment:
I used this form to minimize changes, but it could be replaced an actual `class` definition if desired.

Reply:
Yeah, that makes more sense. I don't see a reason a "singleton" was done this way, it's enough to call the class `_MissingType` to indicate it's not public.

Reply:
Always the challenge when interacting with unfamiliar projects/maintainers around finding the balance of "minimize changes" vs "if you were building this today, it would look very different" :laughing: